cclib/parser/psi4parser.py

```diff
@@ -707,26 +707,29 @@ def get_normalization_factor(exp, lx, ly, lz):
         # a paritcular target it means they are not used (marked also with an 'o'), and in this case
         # we will set a value of numpy.inf so that any value will be smaller.
         #
-        #  ==> Convergence Check <==
-        #
+        #                               ==> Convergence Check <==                                  
+        #  
         #  Measures of convergence in internal coordinates in au.
+        #  
         #  Criteria marked as inactive (o), active & met (*), and active & unmet ( ).
-        #  ---------------------------------------------------------------------------------------------
-        #   Step     Total Energy     Delta E     MAX Force     RMS Force      MAX Disp      RMS Disp
-        #  ---------------------------------------------------------------------------------------------
-        #    Convergence Criteria    1.00e-06 *    3.00e-04 *             o    1.20e-03 *             o
-        #  ---------------------------------------------------------------------------------------------
-        #      2    -379.77675264   -7.79e-03      1.88e-02      4.37e-03 o    2.29e-02      6.76e-03 o  ~
-        #  ---------------------------------------------------------------------------------------------
+        #  
+        #  ----------------------------------------------------------------------------------------------
+        #     Step    Total Energy     Delta E     Max Force     RMS Force      Max Disp      RMS Disp
+        #  ----------------------------------------------------------------------------------------------
+        #    Convergence Criteria     1.00e-06 *    3.00e-04 *             o    1.20e-03 *             o
+        #  ----------------------------------------------------------------------------------------------
+        #       2    -379.77675264   -7.79e-03      1.88e-02      4.37e-03 o    2.29e-02      6.76e-03 o  ~
+        #  ----------------------------------------------------------------------------------------------
         #
+        
         if (self.section == "Convergence Check") and line.strip() == "==> Convergence Check <==" \
             and not hasattr(self, 'finite_difference'):
 
             if not hasattr(self, "optstatus"):
                 self.optstatus = []
             self.optstatus.append(data.ccData.OPT_UNKNOWN)
 
-            self.skip_lines(inputfile, ['b', 'units', 'comment', 'dash+tilde', 'header', 'dash+tilde'])
+            self.skip_lines(inputfile, ['b', 'units', 'b', 'comment', 'b', 'dash+tilde', 'header', 'dash+tilde'])

```

Comment:
Ok, so the header changed to include two new blank lines as well. Since this will break older versions of Psi4, you can add a version-specific conditional:
```python
from packaging.version import parse as parse_version
from packaging.version import Version
<snip>
self.skip_lines(inputfile, ['b', 'units'])
if packaging.version.parse(self.metadata["package_version"]) >= Version("1.7"):
    self.skip_line(inputfile, 'b')
self.skip_line(inputfile, 'comment')
if packaging.version.parse(self.metadata["package_version"]) >= Version("1.7"):
    ...
self.skip_lines(inputfile, ['dash+tilde', ...])
```
This isn't efficient but the logic should be right.